relayer/src/foreign_client.rs

```diff
@@ -696,37 +698,28 @@ impl<DstChain: ChainHandle, SrcChain: ChainHandle> ForeignClient<DstChain, SrcCh
         }
     }
 
-    // Validate a non-zero trusted height to make sure that there is a corresponding
-    // consensus state at the given trusted height on the destination chain's client.
+    /// Validate a non-zero trusted height to make sure that there is a corresponding
+    /// consensus state at the given trusted height on the destination chain's client.
     fn validate_trusted_height(
         &self,
         target_height: Height,
         trusted_height: Height,
         client_state: &AnyClientState,
     ) -> Result<(), ForeignClientError> {
-        if client_state.latest_height() == trusted_height {
-            Ok(())
-        } else {
+        if client_state.latest_height() != trusted_height {
             // There should be no need to validate a trusted height in production,
             // Since it is always fetched from some client state. The only use is
             // from the command line when the trusted height is manually specified.
             // We should consider skipping the validation entirely and only validate
             // it from the command line itself.
 
             warn!("[{}] validating that trusted height {} for target height {} is present in the full list of consensus state heights; this may take a while",
```

Comment:
I think we need to remove this warning message, since it does not apply anymore.
```suggestion
```